apidoc/Map.yml

```diff
@@ -547,3 +555,64 @@ examples:
             }
 
             $.index.open();
+
+  - title: Map Example With Marker Annotation and Clustering
+    example: |
+        This is a map example which creates marker annotation and clustering of annotations.
+        
+            var Map = require('ti.map');
+  
+            var win = Titanium.UI.createWindow({
+                            fullscreen : true,
+                        });
```

Comment:
We have some issues in this example:

- Use 2-space intentation (sometimes it currently has like 12, sometimes 0)
- Do not use the `fullscreen` property as it may confuse people

Reply:
Done.